codebuild/spec/buildspec_unit_nix.yaml

```diff
@@ -0,0 +1,167 @@
+        image: public.ecr.aws/l1b2r3y5/nix-aws-codebuild-aarch64:next
+        privileged-mode: true
+        type: ARM_CONTAINER
+        variables:
+          NIXDEV_ARGS: --ignore-environment --max-jobs auto .#openssl111
+          NIX_CACHE_BUCKET: s3://s2n-tls-nixcachebucket-aarch64?region=us-west-2
+          S2N_NO_HEADBUILD: "1"
+          S2N_TEST: unit
+
+    # Openssl102 aarch64
+    - identifier: UnitOpenssl102_aarch64
+      depend-on:
+        - nixCache_aarch64
+      env:
+        compute-type: BUILD_GENERAL1_LARGE
+        image: public.ecr.aws/l1b2r3y5/nix-aws-codebuild-aarch64:next
+        privileged-mode: true
+        type: ARM_CONTAINER
+        variables:
+          NIXDEV_ARGS: --ignore-environment --max-jobs auto .#openssl102
+          NIX_CACHE_BUCKET: s3://s2n-tls-nixcachebucket-aarch64?region=us-west-2
+          S2N_NO_HEADBUILD: "1"
+          S2N_TEST: unit
+
+    # awslc x86
+    - identifier: UnitAwslc_x86_64
+      depend-on:
+        - nixCache_x86_64
+      env:
+        compute-type: BUILD_GENERAL1_LARGE
+        image: public.ecr.aws/l1b2r3y5/nix-aws-codebuild:latest
+        privileged-mode: true
+        variables:
+          NIXDEV_ARGS: --ignore-environment --max-jobs auto .#awslc
+          NIX_CACHE_BUCKET: s3://s2n-tls-nixcachebucket-x86-64?region=us-west-2
+          S2N_NO_HEADBUILD: "1"
+          S2N_TEST: unit
+    # awslc aarch64
+    - identifier: UnitAwslc_aarch64
+      depend-on:
+        - nixCache_aarch64
+      env:
+        compute-type: BUILD_GENERAL1_LARGE
+        image: public.ecr.aws/l1b2r3y5/nix-aws-codebuild-aarch64:next
+        privileged-mode: true
+        type: ARM_CONTAINER
+        variables:
+          NIXDEV_ARGS: --ignore-environment --max-jobs auto .#awslc
+          NIX_CACHE_BUCKET: s3://s2n-tls-nixcachebucket-aarch64?region=us-west-2
+          S2N_NO_HEADBUILD: "1"
+          S2N_TEST: unit
+    # awslcfips 2022 aarch64
+    - identifier: UnitAwslcFips2022_aarch64
+      depend-on:
+        - nixCache_aarch64
+      env:
+        compute-type: BUILD_GENERAL1_LARGE
+        image: public.ecr.aws/l1b2r3y5/nix-aws-codebuild-aarch64:next
+        privileged-mode: true
+        type: ARM_CONTAINER
+        variables:
+          NIXDEV_ARGS: --ignore-environment --max-jobs auto .#awslcfips2022
+          NIX_CACHE_BUCKET: s3://s2n-tls-nixcachebucket-aarch64?region=us-west-2
+          S2N_NO_HEADBUILD: "1"
+          S2N_TEST: unit
+    # awslcfips 2024 aarch64
+    - identifier: UnitAwslcFips2024_aarch64
+      depend-on:
+        - nixCache_aarch64
+      env:
+        compute-type: BUILD_GENERAL1_LARGE
+        image: public.ecr.aws/l1b2r3y5/nix-aws-codebuild-aarch64:next
+        privileged-mode: true
+        type: ARM_CONTAINER
+        variables:
+          NIXDEV_ARGS: --ignore-environment --max-jobs auto .#awslcfips2024
+          NIX_CACHE_BUCKET: s3://s2n-tls-nixcachebucket-aarch64?region=us-west-2
+          S2N_NO_HEADBUILD: "1"
+          S2N_TEST: unit
+
```

Comment:
Is S2N_TEST necessary? Isn't that leftover from s2n_codebuild.sh?

Reply:
no, removed